examples/kitchen-sink.yaml

```diff
@@ -15,6 +15,11 @@ file_format: "0.1"
 # Environment variable: OTEL_SDK_DISABLED
 disabled: false
 
+# Configure the log level used by the SDK logger
+#
+# Environment variable: OTEL_LOG_LEVEL
+log_level: info
+
```

Comment:
do you think it makes sense to add a top-level `sdk` node in order to scope the SDK configuration now that we also support `instrumentation` config?

Reply:
I kinda think of the root of the doc as the SDK's node. I could be convinced otherwise if folks feel strongly. I'd suggest discussing in a separate issue

Reply:
`disabled` is an SDK property, and at the top level, so this is consistent as is.
LGTM.